Description: 7130140: using horizontal scroll button on mouse causes a message to be printed on stdout
Reviewed-by: art
Origin: http://hg.openjdk.java.net/jdk8/jdk8/jdk/rev/19431d07bc19
Bug-Debian: http://bugs.debian.org/679036
Bug-Vendor: http://bugs.sun.com/bugdatabase/view_bug.do?bug_id=7130140
--- openjdk/jdk/src/share/classes/java/awt/event/MouseEvent.java.orig
+++ openjdk/jdk/src/share/classes/java/awt/event/MouseEvent.java
@@ -758,7 +758,6 @@
 
             if (getModifiersEx() != 0) { //There is at least one more button in a pressed state.
                 if (id == MouseEvent.MOUSE_RELEASED || id == MouseEvent.MOUSE_CLICKED){
-                    System.out.println("MEvent. CASE!");
                     shouldExcludeButtonFromExtModifiers = true;
                 }
             }
